common/supplemental/supplementalData.xml

```diff
@@ -2783,7 +2783,8 @@ XXX Code for transations where no currency is involved
 		<territory type="CN" gdp="25360000000000" literacyPercent="95.1" population="1394020000">	<!--China-->
 			<languagePopulation type="zh" populationPercent="90" officialStatus="official"/>	<!--Chinese-->
 			<languagePopulation type="wuu" populationPercent="6"/>	<!--Wu Chinese-->
-			<languagePopulation type="yue_Hans" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
+			<languagePopulation type="yue" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
+			<languagePopulation type="yue_Hant" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Traditional)-->
```

Comment:
These should be the following, to correct one of the errors. 

```suggestion
			<languagePopulation type="yue_Hans" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Simplified)-->
			<languagePopulation type="yue" populationPercent="5.2" references="R1219"/>	<!--Cantonese (Traditional)-->
```

That is, the current defaults (which we are not changing yet) are:

yue has as default script Hant // in other words, if in isolation, yue_Hant == yue
yue_CN has as default script Hans // in other words, yue_Hans_CN == yue_CN

So yue_Hant normalizes to yue in the supplemental data.

The defaults are set by defaultContents and by likely subtags in the supplemental data, but we shouldn't need to mess with those for this ticket. 

See if that fixes the error, otherwise I'll look into it.